Cargo.toml

```diff
@@ -8,22 +8,19 @@ authors = ["The ChromiumOS Authors"]
 edition = "2021"
 
 [features]
-default = ["vaapi"]
-vaapi = ["libva"]
-v4l2 = ["v4l2r"]
+default = []
+vaapi = ["libva", "anyhow", "byteorder", "thiserror", "crc32fast", "nix"]
```

Comment:
Are you planning to remove these dependencies for `vaapi` and `v4l2` in a later change?

Reply:
Probably not in the near future. These crates are already added to ChromeOS and Android. I'm mostly concerned about dependencies for Chrome, and we're only using the parsers in Chrome for the time being.